src/components/beat.js

```diff
@@ -875,11 +876,11 @@ AFRAME.registerComponent('beat', {
 			this.returnToPool(true);
 		}
 
-		if (!settings.settings.noEffects) {
-			this.explodeEventDetail.position.copy(this.el.object3D.position);
-			this.explodeEventDetail.rotation.copy(this.randVec);
-			mineParticles.emit('explode', this.explodeEventDetail, false);
-		}
+		// if (!settings.settings.noEffects) {
+		// 	this.explodeEventDetail.position.copy(this.el.object3D.position);
+		// 	this.explodeEventDetail.rotation.copy(this.randVec);
+		// 	mineParticles.emit('explode', this.explodeEventDetail, false);
+		// }
```

Comment:
There were 2 errors occuring here:

- If `settings.settings.reducedDebris` is `true`, then `this.randVec` is `undefined`
- The `mineParticles` element does not exist since e9dbf765a096b31fba280f092af6ae8e14d6f9ed